tests/vmi_lifecycle_test.go

```diff
@@ -1066,43 +1101,32 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 					Cores: 1,
 					Features: []v1.CPUFeature{
 						{
-							Name:   "lahf_lm",
+							Name:   "apic",
 							Policy: "require",
 						},
 						{
-							Name:   "mmx",
+							Name:   "fpu",
 							Policy: "disable",
 						},
```

Comment:
should we be calculating features here based on what's present or not present on the compute nodes in the cluster?